### ISSUE TITLE
android auto

### ISSUE THREAD

**Issue by zenusiek** [LawPaul/MH2p_SD_ModKit#1]
Any chances of creating a mod to activate android auto on pcm5 if there's only wired carplay? Thanks!

**Comment by LawPaul** [comment]
MH2p Toolbox (https://www.drive2.ru/b/688240678869690041/) has an "enable all FECs" option that activates Android Auto. That script can be easily ported to this ModKit. I don't have a good way to confirm it is working correctly.

**Comment by zenusiek** [comment]
> MH2p Toolbox (https://www.drive2.ru/b/688240678869690041/) has an "enable all FECs" option that activates Android Auto. That script can be easily ported to this ModKit. I don't have a good way to confirm it is working correctly.

Yeah, I was trying to learn about the toolbox but failed due to lack of necessary hardware. I'm not sure what should be transfered from the toolbox into ModKit. But I did carplay full screen using borrowed iphone and works great! Still I don't want to use ihpone :)

**Comment by LawPaul** [comment]
Send me a message on one of the other platforms I'm on (in the README) and I can send you that script ported to the SD ModKit. I don't want to post it yet as I do not have a way to verify it works properly myself.

**Comment by Dimitri2g** [comment]
> Send me a message on one of the other platforms I'm on (in the README) and I can send you that script ported to the SD ModKit. I don't want to post it yet as I do not have a way to verify it works properly myself.

Could you send it to me, I might be able to get it working. Well what I mean is I will take a wack at it and if I get things working I will add documentation to your repo and make the necessary updates.

**Comment by LawPaul** [comment]
> > Send me a message on one of the other platforms I'm on (in the README) and I can send you that script ported to the SD ModKit. I don't want to post it yet as I do not have a way to verify it works properly myself.
> 
> Could you send it to me, I might be able to get it working. Well what I mean is I will take a wack at it and if I get things working I will add documentation to your repo and make the necessary updates.

Unfortunately, it is not as simple as directly porting the script. 5F module flashing requires a “persistent reboot” which does not happen during a software update. Also, the toolbox uses a modified binary that I cannot redistribute. I am working on alternatives.

**Comment by LawPaul** [comment]
Android Auto mod has been created.

**Comment by Dimitri2g** [comment]
I will give a go.
